docs/testing-with-codeception.md

```diff
@@ -744,3 +745,20 @@ Projects can use a custom Codeception configuration file and override Altis' zer
 ```sh
 composer dev-tools codecept run -- -c path/to/codeception.yml
 ```
+
+## FAQ
+
+- Why do my tests fail because it cannot find the content I created using WPBrowser DB helper functions ?
+
+WPBrowser DB helper functions, like `$I->havePostInDatabase()` and the alike, uses direct database queries to manage content, which means WordPress filters are not run for those operations, so integrations like ElasticPress are not notified of the changes and do not update the Elastic index as a result of that. So while the content is created in the database, it is not synced to Elastic index, and subsequently will not show up in queries that are typically run by Elastic.
+
+The fix for that would be to explicitly reindex content after such direct database operations to ensure the Elastic index is synced properly, and for that we have `$I->reindexContent()` helper function that does that for you.
+
+Example:
+
+```php
+$I->havePostInDatabase( $params );
+$I->haveUserInDatabase( $params );
+
+$I->reindexContent();
```

Comment:
Can we document the parameter it takes so folks know you could pass additional arguments like `--indexables` etc?

Reply:
```suggestion
// Use $extra_params to pass params to the `elasticpress` CLI command like `--indexables=post,user`, etc.
$I->reindexContent( $extra_params );
```